triton/model_repo/feature_extractor/1/model.py

```diff
@@ -0,0 +1,144 @@
+import triton_python_backend_utils as pb_utils
```

Comment:
Do you need to install https://github.com/triton-inference-server/python_backend?
Its readme.md says you have to install it in the docker image.

Correct me if I am wrong.

Reply:
I think we don't need to install it. Its readme looks like install a python_backend example. Just tritonserver official docker could offer all triton ensentials we need.